test/e2e/push_test.go

```diff
@@ -371,7 +371,7 @@ var _ = Describe("Podman push", func() {
 	It("podman push to docker-archive", func() {
 		SkipIfRemote("Remote push does not support docker-archive transport")
 		tarfn := filepath.Join(podmanTest.TempDir, "alp.tar")
-		session := podmanTest.Podman([]string{"push", "-q", ALPINE,
+		session := podmanTest.Podman([]string{"push", "-q", ALPINE, "--compression-format=gzip",
```

Comment:
Does this require a change in podman or contianers/image?

Reply:
This is specific to `containers.conf`. So not c/image; Podman or maybe c/common/libimage.